src/lib.rs

```diff
@@ -150,10 +190,17 @@
 #![warn(missing_docs)]
 #![cfg_attr(feature = "nightly", feature(panic_handler))]
 
+#![cfg_attr(all(not(feature = "use_std"), not(test)), no_std)]
+
+#[cfg(all(not(feature = "use_std"), not(test)))]
+#[macro_use]
```

Comment:
This macro use should be ok to omit because the compiler will inject `#[macro_use] extern crate core`